src/ghga_service_commons/api/api.py

```diff
@@ -217,6 +219,29 @@ async def correlation_id_middleware(
         return response
 
 
+async def request_logging_middleware(request: Request, call_next):
+    """Measure and log the amount of time it takes to process the HTTP request."""
+    url = request.url
+    start_time = time.time()
+    response = await call_next(request)
+    duration = round((time.time() - start_time) * 1000)
+    try:
+        status_phrase = http.HTTPStatus(response.status_code).phrase
+    except ValueError:
+        status_phrase = ""
+    msg = f'{request.method} {url} "{response.status_code} {status_phrase}" - {duration} ms'
+    log.info(
+        msg,
+        extra={
+            "method": request.method,
+            "url": str(url),
+            "status_code": response.status_code,
+            "duration": duration,
```

Comment:
```suggestion
            "duration_ms": duration,
```

so that the unit is clear from the json